src/controller/python/chip/clusters/attribute.cpp

```diff
@@ -370,6 +370,16 @@ PyChipError pychip_WriteClient_WriteAttributes(void * appContext, DeviceProxy *
     client.release();
     callback.release();
 
+    if (busyWaitMs) {
+        auto durationInMs = chip::System::Clock::Milliseconds32(busyWaitMs);
+        auto & clock = chip::System::SystemClock();
+        auto start   = clock.GetMonotonicTimestamp();
+        while (clock.GetMonotonicTimestamp() - start < durationInMs)
+        {
+            // nothing to do.
```

Comment:
Do you need to busy-wait, or can you just `usleep()`?

Reply:
That works! Thanks